### PR TITLE
Patch Initiative rolls out of Lucky Feat

### PR DESCRIPTION
**Pull request by Turtlelord26**
Now enumerated use cases for Lucky, not including Initiative, per Issue #167 . Wording touched up to match enumeration.

I specifically ask for the sake of Luck >= 8 whether we can introduce the phrase Roll with Advantage to CX.

### CODE REVIEW

**Comment by 1sourcecontrol** [comment]
>I specifically ask for the sake Luck >= 8 whether we can introduce the phrase Roll with Advantage to CX.
@Turtlelord26 

Can you rephrase your sentence? I think you are asking if we can do, ">=" on stuff, but I want to be sure.
(I see no harm in using "<=" or ">="  )

**Comment by Turtlelord26** [comment]
I specifically ask, for the sake of Luck >= 8 ^cases^, **whether we can introduce the phrase Roll with Advantage** to CX. @1sourcecontrol

**Comment by bleehu** [comment]
@1sourcecontrol I like the idea of adding "roll with advantage" and "roll with disadvantage" to CX. While I'm quite proud of how different CX is from DnD, I'm not so proud that I won't steal what's good from them. And I'm pretty sure there's no legal reason not to.

**Review by bleehu — APPROVED** [review]
shipit!

**Comment by Turtlelord26** [comment]
@Wildlovelies As the originator of the Issue I'll be waiting on word from you on these changes.

File: Basic Rules.txt [hunk 1]
```diff
@@ -43,6 +43,14 @@ which dice represents the tens place before you roll. i.e. if one die rolls a
 die which represents your ten's place and a purple die which represents your one's
 place, and you roll an orange 3 and a purple 9 then you rolled 39%.
 
+Sometimes circumstances or the occasional feat may prompt you to make a roll with
```
Comment (1sourcecontrol) [inline Basic Rules.txt]: Can you add a header to this so we can easily find the rule? Such as:

>==== Rolling with Advantage/Disadvtange====

or

>Rolling with Advantage/Disadvantage:

Comment (bleehu) [reply Basic Rules.txt]: Bleehu Likes this post!

File: CharacterCreation/03_Feats.txt [hunk 2]
```diff
@@ -545,29 +545,28 @@ Lucky:
...
 	Luck = 9		
-		You can choose to reroll a die, three times per long rest. You can only
-		reroll once per separate die use and if you reroll, choose which roll to keep.
+		You can choose to roll a check, save, or attack with advantage three times 
+		per long rest.
 	Luck = 10		
-		You can choose to reroll a die, four times per long rest. You can reroll
-		as many times as you want (which uses up your rerolls-per-rest) and if you 
-		reroll, choose which of the rolls to keep.
+		You may choose to reroll a check, save, or attack four times per long rest.
```
Comment (1sourcecontrol) [inline CharacterCreation/03_Feats.txt:560]: Isn't it supposed to be "role with advantage", to keep the trend in terminology?

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]

File: CharacterCreation/03_Feats.txt (hunk 2 above)
Comment (bleehu) [reply CharacterCreation/03_Feats.txt:560]: Reroll means you must take the second roll regardless of whether the second roll is superior or not. I don't see the need to formally define reroll, since it's a word in the English language.

Comment (Turtlelord26) [reply CharacterCreation/03_Feats.txt:560]: The problem with using Advantage specifically in this spot is that in that, with advantage terminology, this subfeat is allowing you to choose whether to make a roll with no advantage, advantage, double advantage, or triple advantage, and your choice makes a difference to how many times you can do it again. The original phrasing is more clear.

Comment (1sourcecontrol) [reply CharacterCreation/03_Feats.txt:560]: @bleehu the wording here is supposed to be saying a fancy kind of "rolling with advantage"... unless you meant to clarify something else?

@Turtlelord26 , thanks, that makes sense.

**Review by 1sourcecontrol — APPROVED** [review]

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
oops, I approved without checking to see if you added the header

File: CharacterCreation/03_Feats.txt (hunk 2 above)
Comment (bleehu) [reply CharacterCreation/03_Feats.txt:560]: @1sourcecontrol 
Reroll:
take the 2nd roll
Advantage:
take the higher roll

**Review by 1sourcecontrol — APPROVED** [review]
